src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Altinn.Authorization.ABAC" Version="0.0.3-alpha" />
+    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="1.3.8" />
```

Comment:
storage interface should be updated to the latest version. I think it's 2.0.0

Reply:
removed the nuget package because it was not used